config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,16 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [:password, :age]
+
+WHITELISTED_KEYS_MATCHER = /((^|_)ids?|action|controller|code|created_at|updated_at|recorded_at|deleted_at|limit|process_token$)/.freeze
```

Comment:
Shall we call it an ALLOWLIST or SAFELIST?

Reply:
Its now in a module and the naming is improved.

Reply:
Hey @rsanheim, I think @kitallis was referring to moving away from "whitelist/blacklist" terminology to use "allowlist/denylist" instead. Similar to moving away from "master/slave" towards "primary/follower", etc.

Reply:
Sorry, I missed that completely. Yup, I'll change it to allow-list or something similar.